### ISSUE TITLE
Lizzie 0.6 interface response time on mac is very slow

### ISSUE THREAD

**Issue by mkmatlock** [featurecat/lizzie#510]
I believe this applies mainly to the current master branch.

It seems that the interface response time is extremely slow regardless of whether analysis is running. My machine is not overloaded, as I am running the analysis remotely over ssh. Clicking in the interface, hover to show variation, using any hotkey, or exiting the application typically exhibits a latency around 2-5 seconds. However, title bar display of visits/second updates frequently.

When I run my ssh remote leelaz in a terminal and interact directly with the engine, there is no delayed response.

I am running Mac OSX 10.13.6 on a 2013 macbook pro with java 1.8.0_25

Any suggestions? The old lizzie 0.5 didn't have this problem, but the latest master is the only lizzie that works reliably with my ssh setup and with leelaz 0.16.

**Comment by zsalch** [comment]
If you use the local leelaz, is there the same problem?

**Comment by mkmatlock** [comment]
Yes, but the lag is substantially worse. Title bar still updates the visit/second statistics, but the rest of the UI is basically unusable. I am running leelaz 0.15 locally, with the network weights distributed with the release copy of lizzie 0.6

**Comment by featurecat** [comment]
Did the same issue apply about 10 commits ago? I made some changes during this time that may have affected your performance.

**Comment by featurecat** [comment]
Also please use Leela zero 0.16

**Comment by mkmatlock** [comment]
I updated leela zero on my local machine to 0.16, no improvement. I also did a git revert to revision 71d9265 and re-packaged, but that also did not improve anything.
